### ISSUE TITLE
How to retrain mediapipe hand model 

### ISSUE THREAD

**Issue by Swapnil-gautam** [google/mediapipe#2367]
I want to retrain google's mediapipe hand landmark for more key-points detection but the model is only available in tflite (https://github.com/google/mediapipe/blob/master/mediapipe/modules/hand_landmark/hand_landmark.tflite) format that can not be retrained.
I have created a model identical to mediapipe hand model, I trained it with my custom data (38k images) but facing over-fitting issue, 

I am using RMSprop as optimizer  
MSE (Mean Square Error as loss function
batch size = 32 
initial_learning_rate=1e-3
decay_steps=1000
decay_rate=0.9

![colab](https://user-images.githubusercontent.com/51311257/127965046-2d386d06-2983-4b91-a085-8374318ccc15.png)

**Comment by sgowroji** [comment]
Hi @Swapnil-gautam, Have you tried  Hands solution and which platform are you referring? And what is the error you notice in the above. Thanks!

**Comment by Swapnil-gautam** [comment]
Hey @sgowroji, I have tried Hands solution and it works great but it is only for 21 keypoints and I want more keypoints on fingers for my use case, thus I wanted to perform transfer-learning with mediapipe hand model, but since only tflite model is available and we can not perform transfer-learning with it I create a similar model to media pipe hand and train it with my custom images,
But I am facing over-fitting while training it,

**Comment by sgowroji** [comment]
Duplicate https://github.com/google/mediapipe/issues/2412

**Comment by arianatri** [comment]
@Swapnil-gautam have you reached anywhere ?
